docassemble/ALWeaver/data/questions/assembly_line.yml

```diff
@@ -692,6 +695,20 @@ subquestion: |
 buttons:
   - Restart: restart
 ---
+event: exit_PSEOF_error
+question: |
+  Something is wrong with the format of this PDF
+subquestion: |
+  Docassemble was not able to read ${ template_upload[0].filename }. The error
+  was a "pdfminer.psparser.PSEOF: Unexpected EOF". It might be an XFA form or
+  something else might be wrong with its formatting.
```

Comment:
Don't think that this is important to show the users, and it'll likely confuse them. Should just say we weren't able to read it and to try QPDF. 
```suggestion
  Docassemble was not able to read ${ template_upload[0].filename }.
```

Reply:
This doesn't give any detailed information, though. Is the documate suggestion really the only detail we want to give?

Reply:
IMO, yeah, we should just point them to documate / QPDF (the later is the more helpful one here). If a users comes across this issue, the best case is they do try to use documate / QPDF to fix it, and worst case they give up because they don't understand any of it.  Technical jargon, like "XFA" and "EOF", or the exception message (`pdfminer.psparser.PSEOF: Unexpected EOF`) is going to be confusing, and IMO lead more people to giving up.

If you really want to keep it, maybe using the collapse template with the header "Debugging information". But TBH, if someone came to me with this screen, that exception message wouldn't help me, it'd probably lead me down a rabbit hole trying to figure it out, only to arrive at the same conclusion: just run it through QPDF.